tests/integration/actions/test_metric_list_action.py

```diff
@@ -92,16 +93,17 @@ def execute(self, query, variables=None, headers=None, status_code=None):
 @pytest.fixture
 def local_mercury_db_datasource(
     context: CloudDataContext,
-) -> PostgresDatasource:
+) -> Generator[PostgresDatasource, None, None]:
     datasource_name = "local_mercury_db"
     datasource = context.data_sources.get(name=datasource_name)
+    assert isinstance(datasource, PostgresDatasource)
     yield datasource
 
 
 @pytest.fixture
 def local_mercury_db_organizations_table_asset(
-    local_mercury_db_datasource: PostgresDatasource,
-) -> TableAsset:
+    local_mercury_db_datasource: Any,
```

Comment:
Instead of `Any`, can you match the return type of the fixture above? Maybe you have to change it a  little since it's a Generator.